x-pack/plugins/security/server/index.ts

```diff
@@ -33,6 +33,9 @@ export const config: PluginConfigDescriptor<TypeOf<typeof ConfigSchema>> = {
     rename('sessionTimeout', 'session.idleTimeout'),
     unused('authorization.legacyFallback.enabled'),
   ],
+  exposeToBrowser: {
+    loginAssistanceMessage: true,
+  },
```

Comment:
Unfortunately, no. The `loginAssistanceMessage` value is determined by the actual value provided through configuration (config, env variables etc.) and its schema (defaults, schema references, schema context etc.). In this case core can propagate to the client side exactly what we need.

But `secureCookies` is a bit different, its value is [set to true if TLS is enabled](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security/server/config.ts#L91) and this happens outside of the schema so that Core can't pick this up.

There are a couple of ways to solve this, but I think it deserves its own PR.